scss/gantt/_layout.scss

```diff
@@ -196,7 +196,7 @@
     }
     .k-task-wrap {
         margin: 0 -1.5em;
-        padding: 0 25px;
+        padding: 5px 25px;
```

Comment:
It's related to the size of `k-task-draghandle`. and thus far, we don't have such variable...